frontend/src/components/Auth/FormFormik.tsx

```diff
@@ -13,6 +13,8 @@ type FormProps = {
 };
 
 const FormFormik: React.FC<FormProps> = ({ isSignUp }) => {
+  const [showPassword, setShowPassword] = useState(false);
+
```

Comment:
Change "showPassword" to yes/ no type "question" for booleans (e.g. isPasswordVisible)